sdk/openai/azure-ai-openai/tsp-location.yaml

```diff
@@ -1,5 +1,5 @@
 directory: specification/cognitiveservices/OpenAI.Inference
 additionalDirectories:
     - specification/cognitiveservices/OpenAI.Authoring
-commit: 817861452040bf29d14b57ac7418560e4680e06e
+commit: 0d48109775c587aff0ce83873daa016713f2ff98
```

Comment:
I guess we need to merge [this PR](https://github.com/Azure/azure-rest-api-specs/pull/26699) into `main` so we can update this hash to one from the `main` before proceeding with this PR.

Reply:
Yeah, the spec PR would go first.

I've answered most of the questions (they are good!). Same as you, I'd regret to having "getChatCompletionsWithAzureExtensions" and "beginBeginAzureBatchImageGeneration" get added. But though we can mark them as `internal` in client.tsp, we cannot mark them as `convenientApi(false)` for Java only.